hw/top_darjeeling/lint/top_darjeeling.waiver

```diff
@@ -8,19 +8,43 @@
 set_reset_drivers prim_clock_mux2 prim_flop_2sync prim_flop
 set_clock_drivers prim_clock_buf prim_clock_mux2
 
+# TODO: remove these waivers later when the USB clock was removed from the integrated system
```

Comment:
Please reference an issue number for this TODO item (add it to an existing issue or create one)